### PR TITLE
[release-1.6] :seedling: Refactor test data to align with core CAPI

### PR DESCRIPTION
**Pull request by killianmuldoon**
Signed-off-by: killianmuldoon [kmuldoon@vmware.com](mailto:kmuldoon@vmware.com)

Manual cherry-pick of https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/2250

Part of https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/2251

### CODE REVIEW

**Review by killianmuldoon — COMMENTED** [review]
/test

**Review by killianmuldoon — COMMENTED** [review]
/test pull-cluster-api-provider-vsphere-e2e-full-release-1-6

File: test/integration/integration-dev.yaml [hunk 1]
```diff
@@ -30,53 +30,53 @@ providers:
   - name: cluster-api
     type: CoreProvider
     versions:
-      - name: v1.1.0
+      - name: v1.3.0
         # Use manifest from source files
         value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.0/core-components.yaml"
```
[the remark this answers is not in the document]
Comment (sbueringer) [reply test/integration/integration-dev.yaml:35]: Same below

**Pull request merged**